### ISSUE TITLE
Wrong detection 

### ISSUE THREAD

**Issue by afeena** [mushorg/tanner#49]
When TANNER determine the type of vulnerability, it incorrectly handles xss with links (it marks them as rfi).
For example, these links:
```
'/default.ph/?pageweb=<LINK+REL="stylesheet"+HREF="http://ha.ckers.org/xss.css">'
'/default.ph/?pageweb=<STYLE>@import\\'http://ha.ckers.org/xss.css\\';</STYLE>'
'/default.ph/?pageweb=Redirect+302+/a.jpg+http://victimsite.com/admin.asp&deleteuser'
'/default.ph/?pageweb=<IMG+SRC="http://www.thesiteyouareon.com/somecommand.php?somevariables=maliciouscode">'
'/default.ph/?pageweb=<A+HREF="javascript:document.location=\\'http://www.google.com/\\'">XSS</A>'
'/default.ph/?pageweb=<A+HREF="http://66.102.7.147/">XSS</A>'
'/default.ph/?pageweb=<IFRAME+SRC=http://ha.ckers.org/scriptlet.html+<'
'/default.ph/?pageweb=<META+HTTP-EQUIV="refresh"+CONTENT="0;+URL=http://;URL=javascript:alert(\\'XSS\\');">'
'/default.ph/?pageweb=<XML+SRC="http://ha.ckers.org/xsstest.xml"+ID=I></XML>+<SPAN+DATASRC=#I+DATAFLD=C+DATAFORMATAS=HTML></SPAN>'
```

**Comment by afeena** [comment]
Possible fast fix: change xss order to 3

**Comment by glaslos** [comment]
Can it be only `src` and `href`? We should probably also take the `"` into account.

**Comment by afeena** [comment]
Some examples, there is links without ``"``. I propose to replace order of xss from 2 to 3. It allows to decrease numbers of the wrong detection
``'/default.ph/?pageweb=<LINK+REL="stylesheet"+HREF="http://ha.ckers.org/xss.css">'
'/default.ph/?pageweb=<STYLE>@import\\'http://ha.ckers.org/xss.css\\';</STYLE>'
'/default.ph/?pageweb=Redirect+302+/a.jpg+http://victimsite.com/admin.asp&deleteuser'Exampl
'/default.ph/?pageweb=<IMG+SRC="http://www.thesiteyouareon.com/somecommand.php?somevariables=maliciouscode">'
'/default.ph/?pageweb=<A+HREF="javascript:document.location=\\'http://www.google.com/\\'">XSS</A>'
'/default.ph/?pageweb=<A+HREF="http://66.102.7.147/">XSS</A>'
'/default.ph/?pageweb=<IFRAME+SRC=http://ha.ckers.org/scriptlet.html+<'
'/default.ph/?pageweb=<META+HTTP-EQUIV="refresh"+CONTENT="0;+URL=http://;URL=javascript:alert(\\'XSS\\');">'
'/default.ph/?pageweb=<XML+SRC="http://ha.ckers.org/xsstest.xml"+ID=I></XML>+<SPAN+DATASRC=#I+DATAFLD=C+DATAFORMATAS=HTML></SPAN>'
``

**Comment by glaslos** [comment]
Fixed with dc54ef2
